src/helpers/translation-utils.js

```diff
@@ -0,0 +1,409 @@
+/**
+ * Translation Utilities
+ * 
+ * This utility provides functions to help manage translations in the localization files.
+ * It allows checking for the existence of strings, adding new strings, and updating existing strings.
```

Comment:
FWIW we do have a unit test to check that all strings are avail in all languages.

Out of curiosity, how were you driving this with the actual translated values?

Reply:
I was letting Cline actually drive this...I was not actually using this tool myself but Cline was using it to add strings since it sucked at actually manipulating the large JSON files.  But yeah I went ahead and removed this anyways